tests/invalid/CoCoConvolveNotCorrectlyProvided.nestml

```diff
@@ -26,11 +26,11 @@
 */
 
 neuron CoCoConvolveNotCorrectlyProvided:
-    state:
+    parameters:
```

Comment:
Moving g_ex from ``state`` to ``parameters`` changes the semantics of the test, suggest to move back.

Reply:
Right. Changed the test which will have fewer errors than before since `state` and `intial_values` blocks are merged.